### ISSUE TITLE
BitRefill.Com invoice QR code gives "Unknown link format".

### ISSUE THREAD

**Issue by Ulmo** [LN-Zap/zap-iOS#20]
**Describe the bug**

When I go to BitRefill.Com, and set up an invoice, and scan the QR code, I get:  "Unknown link format".

**To Reproduce**
1. Set up invoice at BitRefill.Com for Lightning Network (mainnet)
2. Zap:  select Send
3. Point at QR code displayed by BitRefill.Com

**Expected behavior**

Continue past that error.  For instance, I was able to copy and paste the link, and then I got the next step, which in my case was another error ("Unable to find a path to ..." which is an LND issue).

**Screenshots**

**Your environment**
- Device: iPhone6+
- OS: iOS 11.4.1
- Version 0.1.0, build 1

**Additional context**

**Comment by Ulmo** [comment]
After I opened up a channel with sufficient pathway, I tried another transaction of the same amount at Bitrefill.  I had the same error from the QR code from Bitrefill, but then I copied the invoice text from Bitrefill and pasted it into https://www.the-qrcode-generator.com/ and then used THAT QR code, and that QR code worked, and I paid my phone bill.

Is there a reason why the QR codes would be different?  Should I show you a sample QR code from Bitrefill, or are you able to pay your phone bill with their QR code to diagnose?  Is it Bitrefill's faulty QR code?
